### PR TITLE
Updated SSH connection setup in protocol guides

### PR DESCRIPTION
**Pull request by SkylineThomasCR**

### CODE REVIEW

File: develop/schemadoc/Protocol/Protocol.Params.Param.Type-options.md [hunk 1]
```diff
@@ -194,22 +194,48 @@ For more information, refer to [Multi-threaded timers SNMP](xref:AdvancedMultiTh
 
 ### ssh pwd
 
-This option specifies that this parameter holds the password used for setting up SSH communication based on user credentials. When using this option, there should also be another parameter that uses the "ssh username" option. The SSH connection is established by DataMiner itself, if the parameters are filled in correctly.
+> [!IMPORTANT]
+> Link this parameter via the [Password](xref:Protocol.PortSettings.SSH.Credentials.Password) element
```
Comment (jaklke) [inline develop/schemadoc/Protocol/Protocol.Params.Param.Type-options.md:198]: I would rewrite this as 
The usage of this option is not recommended. It only support connectors with a single SSH connection using the fixed port 22.  Instead, link this parameter via the [Password](xref:Protocol.PortSettings.SSH.Credentials.Password) element.

File: develop/schemadoc/Protocol/Protocol.Params.Param.Type-options.md [hunk 2]
```diff
@@ -194,22 +194,48 @@ For more information, refer to [Multi-threaded timers SNMP](xref:AdvancedMultiTh
...
+> [!WARNING]
+> Use the [password](xref:Protocol.Params.Param.Measurement.Type-options#options-for-measurement-type-string) option
+> when this parameter is displayed.
+> Using this option ensures greater security.
+
 ### ssh username
 
-This option specifies that this parameter holds the username used for setting up SSH communication based on user credentials. When using this option, there should also be another parameter that uses the "ssh pwd" option. The SSH connection is established by DataMiner itself, if the parameters are filled in correctly.
+> [!IMPORTANT]
+> Link this parameter via the [Username](xref:Protocol.PortSettings.SSH.Credentials.Username) element
```
Comment (jaklke) [inline develop/schemadoc/Protocol/Protocol.Params.Param.Type-options.md:217]: I would rewrite this as 
he usage of this option is not recommended. It only support connectors with a single SSH connection using the fixed port 22. Instead, link this via the [Username](xref:Protocol.PortSettings.SSH.Credentials.Username) element

File: develop/schemadoc/Protocol/Protocol.Params.Param.Type-options.md [hunk 3]
```diff
@@ -194,22 +194,48 @@ For more information, refer to [Multi-threaded timers SNMP](xref:AdvancedMultiTh
...
+When using this option, there should also be another parameter that uses the "ssh pwd" option. 
+The SSH connection is established by DataMiner itself, if the parameters are filled in correctly.
 
 Only applicable for parameters of type read.
 
 ### ssh options
 
-This option specifies that this parameter holds the required data for setting up SSH communication based on an identity file. This is an alternative way to set up an SSH connection (instead of user credentials). The content of the "ssh options" parameter is as follows:
+> [!IMPORTANT]
+> Link this parameter via the [Identity](xref:Protocol.PortSettings.SSH.Identity) element
```
Comment (jaklke) [inline develop/schemadoc/Protocol/Protocol.Params.Param.Type-options.md:231]: The usage of this option is not recommended. It only support connectors with a single SSH connection using the fixed port 22. Instead, link this via the [Identity](xref:Protocol.PortSettings.SSH.Identity) element.

File: develop/schemadoc/Protocol/Protocol.Params.Param.Type-options.md (hunk 1 above)
Comment (SkylineThomasCR) [reply develop/schemadoc/Protocol/Protocol.Params.Param.Type-options.md]: Applied feedback

File: develop/schemadoc/Protocol/Protocol.Params.Param.Type-options.md (hunk 2 above)
Comment (SkylineThomasCR) [reply develop/schemadoc/Protocol/Protocol.Params.Param.Type-options.md]: Applied feedback

**Review by MariekeGO — APPROVED** [review]

**Pull request merged**